packages/presentation/lib/screen/movie_details/details_bloc.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:domain/use_case/get_people_use_case.dart';
+import 'package:presentation/base/bloc.dart';
+import 'package:presentation/screen/movie_details/details_data.dart';
+import 'package:presentation/screen/movie_details/details_screen.dart';
+
+abstract class DetailsBloc extends Bloc<DetailsScreenArguments, DetailsData> {
+  factory DetailsBloc(
+    GetPeopleUseCase getPeopleUseCase,
+  ) =>
+      DetailsBlocImpl(
+        getPeopleUseCase,
+      );
+}
+
+class DetailsBlocImpl extends BlocImpl<DetailsScreenArguments, DetailsData>
+    implements DetailsBloc {
+  final GetPeopleUseCase _blocGetCast;
+
+  DetailsBlocImpl(
+    this._blocGetCast,
+  );
+
+  DetailsData _stateData = DetailsData.init();
```

Comment:
if you are injecting variables above the constructor then keep doing that

Reply:
corrected